packages/player-component/src/dotlottie-player.ts

```diff
@@ -281,6 +300,42 @@ export class DotLottiePlayer extends LitElement {
     return this._dotLottieCommonPlayer.animations.size;
   }
 
+  /**
+   * @returns The ids of all the animations
+   */
+  public animations(): string[] {
+    if (!this._dotLottieCommonPlayer) return [];
+
+    return Array.from(this._dotLottieCommonPlayer.animations.keys());
+  }
+
+  /**
+   * @returns The ids of all the styles
+   */
+  public styles(): string[] {
+    if (!this._dotLottieCommonPlayer) return [];
+
+    return Array.from(this._dotLottieCommonPlayer.themes.keys());
+  }
+
+  /**
+   * @returns The current applied theme
+   */
+  public currentTheme(): string {
+    if (!this._dotLottieCommonPlayer) return '';
+
+    return this._dotLottieCommonPlayer.defaultTheme;
+  }
```

Comment:
if we are going to give a different name to the defaultTheme, I think we should also do it in the player core for consistency

```suggestion
  public defaultTheme(): string {
    if (!this._dotLottieCommonPlayer) return '';

    return this._dotLottieCommonPlayer.defaultTheme;
  }
```

Reply:
Have renamed to "getDefaultTheme" as defaultTheme clashed with the prop name and I cant define

get defaultTheme